src/SupervisionSuccess/model/transformData.js

```diff
@@ -0,0 +1,73 @@
+import calcRevocations from "./calcRevocations";
+import calcSavings from "./calcSavings";
+
+/* Number of months to add to the end of chart (chart need to be continued),
+This months should not participate in savings and prisonPopulationDiff calculations
+*/
+const ADDED_MONTHS = 5;
+
+/**
+ * Function that transforms params and app state to chartData and stats
+ * @param {{
+     newOffensePopulation: number
+     revocationA: number
+     revocationsTimescale: number
+     savingsMap: {checkpoint: number savings: number}[]
+     marginalCostPerInmate: number
+   }} params - params for chosen state taken from source file
+ * @param {number} implementationPeriod
+ * @param {number} projections
+ * @param {number} changeInRevocations
+ * @returns {{chartData: {month: number, totalPopulation: number, baseline: number}[], prisonPopulationDiff: number, savings: number}}
+ */
+function transformData(params, implementationPeriod, projections, changeInRevocations) {
```

Comment:
I think `transformData` may be a bit too vague. Can we call it something like `produceProjection`?

Reply:
Sure, have been thinking about this but nothing descriptive did not came to mind.